### ISSUE TITLE
Retrain w/ own Dataset: Choice of Parameters

### ISSUE THREAD

**Issue by syikmis** [qqwweee/keras-yolo3#587]
Yeehawww guys and girls,
has someone successfully retrained the model w/ another dataset then COCO or similar? 
I am retraining the model w/ Crowdhuman dataset for 50 epochs with frozen layers `optimizer=Adam(lr=3e-3)` and then another 50 epochs for all layers `optimizer=Adam(lr=1e-3)` - just like it is in train.py. 
Training takes guite time (15k images), so i am not able to play around with various parameters.

After 60 epochs of training i have a val_loss = 95.39, which is obviously not very good..

Has someone suggestions for an improved training procedure or better parametervalues?

I already use the anchors produced by the k-means algorithm and got an avg IoU of 69,45%.

Thx very much!

**Comment by tabmoo** [comment]
Could you please share the annotations in the format of this repository so I could try Crowdhuman myself and compare the loss with yours?
